src/main/webapp/search_area.js

```diff
@@ -120,53 +150,92 @@ class SearchArea {
       });
 
       newOrganization.closeButtonElement.addEventListener('organization-close', () => {
-      //  Remove the popup from the DOM.
+        //  Remove the popup from the DOM.
         document.getElementById("organization-popup-area").classList.add("hide-popup");
         document.getElementById("organization-popup-area").classList.remove("show-popup");
         newOrganization.popupElement.remove();
       });
 
-      this.organizationListArea.appendChild(newOrganization.getOrganization());
+      this.listArea.appendChild(newOrganization.getOrganization());
     });
     /* If the query has returned 0 organization objects, display No Results Found message */
-    if ((this.organizationObjectsList.length === 0) && (this.organizationListArea.innerHTML === '')) {
+    if ((this.objectsList.length === 0) && (this.listArea.innerHTML === '')) {
       const noResultsFoundMessage = document.createElement("div");
       noResultsFoundMessage.setAttribute("id", "no-results-found");
       noResultsFoundMessage.textContent = "No results found for current filters.";
-      this.organizationListArea.appendChild(noResultsFoundMessage);
+      this.listArea.appendChild(noResultsFoundMessage);
     }
   }
 
   async getListOfOrganizations() {
     const response = await fetch(`/list-organizations?${this.filterParams.toString()}`);
-    this.organizationObjectsList = await response.json();
+    this.objectsList = await response.json();
     const newCursor = await response.headers.get("Cursor");
     this.filterParams.set("cursor", newCursor);
     /* If < 5 results are returned, the end of the given query has been reached */
-    this.lastResultFound = (this.organizationObjectsList.length < 5);
+    this.lastResultFound = (this.objectsList.length < 5);
     if (this.lastResultFound) {
       this.loadMoreButton.classList.add("hide-load-button");
     }
   }
 
+  renderListOfEvents() {
```

Comment:
We can condense the render / get functions into one function for each & just decide which class / fetch link to use inside of those functions instead of line 128

Reply:
we can take care of this when we connect this to the backend though fine to leave as is for this PR

Reply:
I think for reuseablity purposes that they should be separate in the case that we want to render organizations differently than events, thoughts?